pytorch mlp -> main

adke opened the pull request:
This is a work in progress model. I still need to train and test the model for proper accuracy. So I am still trying to figure that out. Therefore, I am aware I am still missing key components to the model.

adke committed:
Work in progress model

adke committed:
Update Read-Me 1

adke committed:
updated to original readme

mushi251 reviewed (commented):
Line 72: I could be wrong, but I believe that self.fc1 is hidden layer#1 and not an input layer. ff so ,you may have to remove self.fc2 

Line 75: Not sure if you need a dropout layer

adke committed:
updated mnist pytorch model

Model is able to train and test along with being able to output the loss and accuracy.
